_releaser/aws.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/pkg/errors"
+)
+
+type AwsCmd struct {
+	UpdateConfig AwsUpdateConfigCmd `kong:"cmd,name=update-config"`
+}
+
+type AwsUpdateConfigCmd struct {
+	BucketName    string `kong:"name='bucket-name',env='AWS_BUCKET_NAME'"`
+	RegionName    string `kong:"name='region-name',env='AWS_REGION_NAME'"`
+	WebsiteConfig string `kong:"name='website-config',env='AWS_WEBSITE_CONFIG'"`
+}
+
+func (s *AwsUpdateConfigCmd) Run() error {
+	file, err := ioutil.ReadFile(s.WebsiteConfig)
+	if err != nil {
+		return errors.Wrapf(err, "failed to read website config file %q", s.WebsiteConfig)
```

Comment:
I guess for the scope of this tool, native error formatting instead of pkg/errors would work (and save a dependency)

But that's a really minor nit / random thought

Reply:
Old habit and I tend to like it :sweat_smile: